src/main/java/com/google/opensesame/github/GitHubGetter.java

```diff
@@ -74,4 +86,17 @@ public static void setGitHubForTests(GitHub mockGitHub) {
   public static void removeGitHubMock() {
     gitHub = null;
   }
+
+  /**
+   * Causes the GitHub API to fail with a RateLimitExceededException when it has exceeded the API
+   * calls alotted to it by GitHub. Without this handler, the API would stall until its available
+   * calls are replenished.
```

Comment:
nit: This wording is a bit confusing imo. Consider:
```suggestion
   * Causes requests made to the GitHub API to fail with a RateLimitExceededException when the API
   * rate limit has been exceeded. Without this handler, requests to the API would hang until the limit
   * has been refreshed.
```